include/sir/platform.h

```diff
@@ -613,20 +613,22 @@ typedef BOOL(CALLBACK* sir_once_fn)(PINIT_ONCE, PVOID, PVOID*);
 
 # endif /* !__WIN__ */
 
-# if (__STDC_VERSION__ >= 201112 && !defined(__STDC_NO_THREADS__)) || \
+# if !defined(_sir_thread_local)
+#  if (__STDC_VERSION__ >= 201112 && !defined(__STDC_NO_THREADS__)) || \
      (defined(__SUNPRO_C) || defined(__SUNPRO_CC))
-#  if defined(_AIX) && defined(__GNUC__)
+#   if defined(_AIX) && defined(__GNUC__)
+#    define _sir_thread_local __thread
+#   else
+#    define _sir_thread_local _Thread_local
+#   endif
+#  elif defined(__WIN__)
+#   define _sir_thread_local __declspec(thread)
+#  elif defined(__GNUC__) || (defined(_AIX) && (defined(__xlC_ver__) || defined(__ibmxl__)))
 #   define _sir_thread_local __thread
 #  else
-#   define _sir_thread_local _Thread_local
-#  endif
-# elif defined(__WIN__)
-#  define _sir_thread_local __declspec(thread)
-# elif defined(__GNUC__) || (defined(_AIX) && (defined(__xlC_ver__) || defined(__ibmxl__)))
-#  define _sir_thread_local __thread
-# else
-#  if !defined(_CH_) && !defined(__CH__)
-#   error "unable to resolve thread local attribute; please contact the author."
+#   if !defined(_CH_) && !defined(__CH__)
+#    error "unable to resolve thread local attribute; please contact the author."
```

Comment:
We should change this and the other messages like it to say 'contact the maintainers'